README.md

```diff
@@ -1,22 +1,36 @@
-# freeipa-master-tf-module #
+# freeipa-server-tf-module #
 
-[![GitHub Build Status](https://github.com/cisagov/freeipa-master-tf-module/workflows/build/badge.svg)](https://github.com/cisagov/freeipa-master-tf-module/actions)
+[![GitHub Build Status](https://github.com/cisagov/freeipa-server-tf-module/workflows/build/badge.svg)](https://github.com/cisagov/freeipa-server-tf-module/actions)
 
 A Terraform module for deploying a FreeIPA server.
 
 ## Usage ##
 
 ```hcl
-module "ipa_master" {
-  source = "github.com/cisagov/freeipa-master-tf-module"
-
-  aws_instance_type   = "t3.large"
-  domain              = "example.com"
-  hostname            = "ipa.example.com"
-  realm               = "EXAMPLE.COM"
-  security_group_ids  = ["sg-51530134", "sg-51530245"]
-  subnet_id           = aws_subnet.master_subnet.id
-  tags                = {
+module "ipa0" {
+  source = "github.com/cisagov/freeipa-server-tf-module"
+
```

Comment:
Since this is showing sample usage, is there a particular reason we no longer have `aws_instance_type`?

Reply:
No reason, other than the fact that I never change it.  Do you want me to add it back in?